tutorials/haskell/golden-testing-in-haskell-with-hspec-gold/code/src/README.md

```diff
@@ -0,0 +1,48 @@
+# Golden testing tutorial
+
+This repository is a guide on how to implement the [hspec-golden](https://github.com/stackbuilders/hspec-golden) library to create Golden tests (or Golden Master Tests) using hspec.
+
+# What are golden tests?
+Golden testing is pretty similar to unit testing, except that the test output is stored in a separate file known as the "Golden File". 
+
+For an in depth explanation refer to this [link](https://www.stackbuilders.com/tutorial/golden-testing-in-haskell-with-hspec-gold).
+
+# How to test this project:
+* **Step 1:** **Download** or **Clone** this repo by console:
+
+``` 
+$ git clone git@github.com:MedAdrian/hspec-golden-tutorial.git
```

Comment:
@MedAdrian Can you update this step to download the code from the `code/` folder?

Reply:
Could you please share an example of this?

Reply:
Unfortunately, `git` doesn't provide a way to do this. However, you can mention `gitzip`: http://kinolien.github.io/gitzip/
```
To download a copy of the code example you can use `gitzip`:

1) Access [gitzip](http://kinolien.github.io/gitzip/)
2) Copy and paste the following URL: https://github.com/stackbuilders/tutorials/tree/tutorials/tutorials/haskell/csv-encoding-decoding/code (**NOTE:** @MedAdrian YOU SHOULD REPLACE IT WITH THE REAL URL AT THE END) in the input to download the sub-folder.
3) Click on download and save the zip somewhere in your file system

Reply:
I think `gitzip` is easy to use, so probably we don't need to specify those steps. I think it would be enough to say:
```
To download a copy of the code example you can use [gitzip](http://kinolien.github.io/gitzip/) pasting the following URL: https://github.com/stackbuilders/tutorials/tree/tutorials/tutorials/haskell/csv-encoding-decoding/code